gluu-server/opt/gluu/jetty/oxauth/custom/scripts/sic/oob.py

```diff
@@ -417,19 +417,22 @@ def changeContact(self, requestParameters):
             identity.setWorkingParameter("oobContact", self.phoneUtil.format(phoneNumber, PhoneNumberUtil.PhoneNumberFormat.NATIONAL))
             identity.setWorkingParameter("manageTask", "oobReplace")
 
-def checkInvalidSession():
+def isManagementAllowed():
```

Comment:
can we document this function a bit (method's doc only), just explain what is meant to do

Reply:
Will do for the next build.